sdk/core/Azure.Core/tests/Azure.Core.Tests.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="Azure.Identity" />
```

Comment:
Shouldn't Azure.Core.Tests still use a ProjectReference?

Reply:
Not to Azure.Identity